### ISSUE TITLE
TF2Caffe successfully but error occur when inference in Caffe

### ISSUE THREAD

**Issue by kelvinou** [microsoft/MMdnn#685]
Platform (ubuntu 16.04):
Python version: 3.6
Source framework with version (Tensorflow 1.12 - CPU):
Pre-trained model  (official examples):

Running scripts:
I use MMdnn given scripts, models such as resnet_v2_152,inception_v3,mobilenet_v1_1.0_frozen,etc.
$ mmdownload -f tensorflow -n resnet_v2_152

I successfully convert the TF model to Caffe by:
$ mmconvert -sf tensorflow -in imagenet_resnet_v2_152.ckpt.meta -iw imagenet_resnet_v2_152.ckpt --dstNodeName MMdnn_Output -df caffe -om tf_resnet

It generates resnet152.prototxt and resnet152.caffemodel.
But error occur when loading them in Caffe inference, and throw an error :
output shape <1000 vs 1001>

And I check out resnet152.prototxt and find that num_output:1001
I try to fix 1001->1000, error still...
Others model also have the same issue.

**Comment by kelvinou** [comment]
resnet152.prototxt
layer {
  name: "resnet_v2_152_logits_Conv2D"
  type: "Convolution"
  bottom: "resnet_v2_152_pool5_reshape"
  top: "resnet_v2_152_logits_Conv2D"
  convolution_param {
    num_output: 1001
    bias_term: true
    group: 1
    stride: 1
    pad_h: 0
    pad_w: 0
    kernel_h: 1
    kernel_w: 1
  }
}
layer {
  name: "MMdnn_Output"
  type: "Reshape"
  bottom: "resnet_v2_152_logits_Conv2D"
  top: "MMdnn_Output"
  reshape_param {
    shape {
      dim: 1001
    }
  }
}
